main.go

```diff
@@ -94,18 +95,8 @@ func main() {
 		log.Fatalf("could not find instance ip address: %s", err)
 	}
 
-	sshKeyPair, err := NewSSHKeyPair(2048)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	sshAgent, err := NewSSHAgent()
-	if err != nil {
-		log.Fatal(err)
-	}
-
 	comment := fmt.Sprintf("%s:%s", config.UserName, instConnect.ID)
-	err = sshAgent.addKey(sshKeyPair.private, comment)
+	session, err := ssh.NewSession(comment)
```

Comment:
Can we name this something like `sshSession`, so it's immediately obvious what it's for?